aiohttp/client.py

```diff
@@ -225,39 +225,35 @@ class ClientTimeout:
 class ClientSession:
     """First-class interface for making HTTP requests."""
 
-    ATTRS = frozenset(
```

Comment:
This part needs to be reverted.

Reply:
what happens to __slots__ ?

Reply:
3.x uses attr instead of slots.

I assume it wasn't backported due to backwards-compatibility concerns. Just keep the 3.x code is it already exists and add the attributes into the frozenset.

Reply:
Ok, I'll do